recipes/medcoupling/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set version = "9.10.0" %}
+
+
+package:
+  name: medcoupling
+  version: '{{ version }}'
+
+source:
+  - fn: 'medcoupling_{{ version }}.tar.gz'
+    url: https://git.salome-platform.org/gitweb/?p=tools/medcoupling.git;a=snapshot;h=V{{ version.replace('.', '_') }};sf=tgz
+    sha256: 02087a4e59ffcdab6fc68460ffd86b03a80b5c0faa695b0bfb2142f1d0d3e5ea
+    patches:
+      - swig_big_file_compile_flag.patch
+  - fn: 'config_{{ version }}.tar.gz'
+    url: https://git.salome-platform.org/gitweb/?p=tools/configuration.git;a=snapshot;h=V{{ version.replace('.', '_') }};sf=tgz
+    folder: deps/config
+    sha256: d58f6749514ce49b00590a96d6dc9090a76a07c13690e2453403643afb40346c
+
+
+build:
+  skip: True  # [osx]
+  detect_binary_files_with_prefix: true
+  # string: py{{ CONDA_PY }}_mpi_{{ mpi }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}   # [mpi != "nompi"]
+  # string: py{{ CONDA_PY }}_nompi_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}           # [mpi == "nompi"]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make                                      # [not win]
+    - ninja                                     # [win]
+    - cmake
+    - swig >=4.0.2
+
+  host:
+    - python
+    - numpy
+    - boost
+    - zlib
+    - libxml2
```

```suggestion
    - hdf5
    - libxml2
```
the static analysis on windows is finding a direct dependency betweeh medcoupling.dll and hdf5. this should be declared
```
 ERROR (medcoupling,Library/bin/medimport.dll): Needed DSO Library/bin/hdf5.dll found in ['conda-forge/win-64::hdf5==1.14.2=nompi_h73e8ff5_100']
  ERROR (medcoupling,Library/bin/medimport.dll): .. but ['conda-forge/win-64::hdf5==1.14.2=nompi_h73e8ff5_100'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
 ```